diff --git a/src/render.c b/src/render.c
index ac1b39c..b9e74e0 100644
--- a/src/render.c
+++ b/src/render.c
@@ -25,11 +25,7 @@ static void render_con_dockarea(Con *con, Con *child, render_params *p);
  * Returns the height for the decorations
  */
 int render_deco_height(void) {
-    int deco_height = config.font.height + 4;
-    if (config.font.height & 0x01) {
-        ++deco_height;
-    }
-    return deco_height;
+    return 0;
 }
 
 /*
