diff --git a/src/render.c b/src/render.c
index 478a1cc4..4644005a 100644
--- a/src/render.c
+++ b/src/render.c
@@ -25,10 +25,7 @@ bool has_adjacent_container(Con *con, direction_t direction);
  * Returns the height for the decorations
  */
 int render_deco_height(void) {
-    int deco_height = config.font.height + 4;
-    if (config.font.height & 0x01)
-        ++deco_height;
-    return deco_height;
+    return 0;
 }
 
 /*
